scheme/sevsnp/evidence_handler.go

```diff
@@ -0,0 +1,408 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package sevsnp
+
+import (
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"time"
+
+	"github.com/google/go-sev-guest/abi"
+	"github.com/google/go-sev-guest/proto/sevsnp"
+	"github.com/google/go-sev-guest/verify"
+	"github.com/google/go-sev-guest/verify/trust"
+	sevsnpParser "github.com/jraman567/go-gen-ref/cmd/sevsnp"
```

Comment:
I need to merge `github.com/jraman567/go-gen-ref` with `github.com/veraison/gen-corim`. It's in my pipeline but has the overhead of refactoring gen-corim.